schema-and-ERD

anmolsingh0005 opened the pull request:
Concept Include in this PR:

1 Schema
2 ER Diagram

[Code Explanation Video](https://www.loom.com/share/18475c3bd8d54c22a4e30b18ef92d382?sid=eabb0330-46b1-43f4-b848-2d504389291e)

anmolsingh0005 committed:
schema-and-ERD

PRASANNAPATHI commented:
Your approved proposal was 'Car Rental System' where users can get a car, with or without driver.
But now, your diagrams and working seems to be simply 'Car Booking' which already done by another student 'Ride share service'.
Change your concept and diagrams as approved with 'Car Rental System' and make necessary attributes and connections.

Apart from that,

What is the need of Transaction Entity here?, and the attribute named Transaction_type - You explained its value as (deposit or withdraw), which is meaningless in your project.
You don't need 'Transaction' Entity, Instead just add payment attributes to the 'Booking' table.

anmolsingh0005 commented:
Hi @PRASANNAPATHI 

Sorry I forgot to inform about the changes i made in my project, the
First one is the transaction table I added newly in it, because my course professor suggested me to add the transaction table in this because I had Booking table in my project.
Second one is the reason I had the changed the Car Rental to Car Booking because the making of it was becoming little complex, So I changed it to Car Booking.

Kalvium-Program commented:
Book a slot in my calendar today, for viva session.

https://calendar.kalvium.com/prasannapathi/30min

PRASANNAPATHI commented:
Your approved idea is 'Car Rental System', you should take up that,
Booking system is already taken by another student.

Book a slot in my calendar today, for viva session.

https://calendar.kalvium.com/prasannapathi/30min
